tests/common.go

```diff
@@ -9295,6 +9322,39 @@ func WaitForSnapShotToReady(snapshotScheduleName, snapshotName, appNamespace str
 
 }
 
+// WaitTillPoolExpanded waits till pool expansion gets completed
+func WaitTillPoolExpanded(poolUUID string, poolCurrSize uint64) error {
```

Comment:
How are you making pool expansion is completed here? Sometimes pool size changes but rebalance might be in-progress

Reply:
yes, you are right , I missed that part , will update it . currently I am checking only till pool size got increased

Reply:
removing this function as we already have waitForPoolToBeResized